### PR TITLE
add splash login signup screens

### PR DESCRIPTION
**Pull request by SeaUrc**
Quicker to apply my changes to new branch than to rebase from master

### CODE REVIEW

File: packages/js/app/src/components/scaleComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Dimensions } from 'react-native';
```
Comment (jbis9051) [inline packages/js/app/src/components/scaleComponent.tsx]: What is scaleComponent and why is it needed?

Comment (SeaUrc) [reply packages/js/app/src/components/scaleComponent.tsx]: The input boxes were too big for aspect ratios like the iphone 8
scale component scales width or height with respect to the width and height of the screen
Should I get rid of it?

Comment (jbis9051) [reply packages/js/app/src/components/scaleComponent.tsx]: CSS *should* be expressive enough to do this, try to do it with just CSS

Comment (SeaUrc) [reply packages/js/app/src/components/scaleComponent.tsx]: I've tried. I will try again

Comment (SeaUrc) [reply packages/js/app/src/components/scaleComponent.tsx]: Sorry, I was being dumb. I am fixing it

**Comment by SeaUrc** [comment]
Should be good now

File: packages/js/app/package.json [hunk 2]
```diff
@@ -20,6 +20,8 @@
         "@react-native-community/masked-view": "*",
         "@react-navigation/bottom-tabs": "6.3.1",
         "@react-navigation/native": "6.0.10",
+        "@react-navigation/native-stack": "^6.7.0",
+        "@rnmapbox/maps": "rnmapbox/maps#main",
```
Comment (jbis9051) [inline packages/js/app/package.json:24]: What is this dependency?

Comment (SeaUrc) [reply packages/js/app/package.json:24]: Used to create the props that allow you to navigate and route

Comment (SeaUrc) [reply packages/js/app/package.json:24]: Ignore my message before if you saw it
You're right, I should just be using @react-navigtion/stack and get rid of unnecessary dependency

**Pull request closed without merging**